sidebars.js

```diff
@@ -12,32 +12,47 @@ module.exports = {
       label: 'Get Started',
       link: { type: 'doc', id: 'index' },
       items: [
+        "get-started/install-okteto-cli",
```

Comment:
> but at this time, you don't have an Okteto instance yet.

I agree, I think this should come under the "For Developers" Category.

> Related to this, I struggle with the "Deploy your first app" tutorial, it's unclear to me if that's a getting started for platform engineers tutorial to a getting started for developers tutorial.

I think it is part of the getting started for Platform Engineers section only because the primary focus in that doc is on writing the Okteto manifest and not the Okteto CLI commands. What we can do is that at the start of that page menetion that you'll need to install Okteto CLI to test out your changes as you write your first Okteto Manifest and then link to the Okteto CLI install page from there. How does that sound?

Reply:
@pchico83 Agree with you feedback but I think we should approach this in a couple iterations.

My suggestion to start is to remove the "configure with your instance" section of the CLI installation guide for now, and have both the developer and the platform onboarding go over this. I think we should do this, add the developer guide, then come back to creating additional pages and improvements in a later PR.

@RinkiyaKeDad Platform engineers need the CLI for managing Okteto so we shouldn't be burying this in the developer section.

Reply:
@codyjlandstrom makes sense. As part of this PR, what about linking to the "Deploy your first app" tutorial in case the git repo the developer uses doesn't have an okteto manifest yet?